app/src/main/java/com/framgia/soundclound/screen/detailgenre/GenreDetailViewModel.java

```diff
@@ -10,9 +10,12 @@
 import com.framgia.soundclound.data.model.Track;
 import com.framgia.soundclound.data.source.TrackDataSource;
 import com.framgia.soundclound.data.source.TrackRepository;
+import com.framgia.soundclound.data.source.local.SharePreferences;
 import com.framgia.soundclound.data.source.remote.TrackRemoteDataSource;
 import com.framgia.soundclound.screen.moretrack.MoreTrackFragment;
+import com.framgia.soundclound.screen.playtrack.PlayTrackActivity;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.screen.playtrack.PlayTrackActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports